catalog/includes/OSC/Sites/Admin/site_conf.php

```diff
@@ -1,9 +1,9 @@
 <?php
 $ini = <<<EOD
-dir_root = ""
-http_server = ""
-http_path = ""
+dir_root = "/var/www/html/admin/"
+http_server = "http://13.57.225.143"
+http_path = "/admin/"
 http_images_path = "images/"
 http_cookie_domain = ""
-http_cookie_path = ""
+http_cookie_path = "/admin/"
```

Comment:
I'm not sure it is a good idea to modify their existing files. We should create a copy and substitute on build

Reply:
Well we can. But these file should come from the user with the dockercompose, Makefile, with their values for the specified fields. And yes the sqldump file should be should be post installation dump. So that it has all these data.

Reply:
So, what's the verdict?